### PR TITLE
Update of FRENCH translation for About us extension

### PR DESCRIPTION
**Pull request by Galixte**
- Update for common.php file.

### CODE REVIEW

File: language/fr/common.php [hunk 1]
```diff
@@ -18,28 +21,44 @@
...
+// translators to re-order the output of data while ensuring it remains correct
+//
+// You do not need this where single placeholders are used, e.g. 'Message %d' is fine
+// equally where a string contains only two placeholders which are used to wrap text
+// in a url you again do not need to specify an order e.g., 'Click %sHERE%s' is fine
+//
+// Some characters you may want to copy&paste:
+// ’ « » “ ” …
+//
+
```
Comment (Crizz0) [inline language/fr/common.php:39]: Do you think this is necassary? 

Why not 
/**
*
*
...
*/

**Comment by Galixte** [comment]
Hi,

yes it’s useful for translators like me. I use this in all my translations.

Take a look on some examples of my translations, there are the same:
- https://github.com/phpbb-extensions/pages/blob/master/language/fr/pages_common.php
- https://github.com/phpbb-extensions/googleanalytics/blob/master/language/fr/googleanalytics_acp.php
- https://github.com/phpbb-extensions/boardannouncements/blob/master/language/fr/boardannouncements.php
- https://github.com/phpbb-extensions/boardrules/blob/master/language/fr/boardrules_common.php
- http://github.com/board3/Board3-Portal/blob/2.1.x/language/fr/modules/portal_donation_module.php
- etc..

**Comment by Crizz0** [comment]
Okay. But then i think i should add these lines to all files, too.

**Comment by Galixte** [comment]
Hi,

i think too. ;)

Could you merge this PR, please ?

**Pull request merged**